app/controllers/programs/students_controller.rb

```diff
@@ -48,6 +47,7 @@ def create
   def destroy
     if @student.reservations.present?
       flash.now[:alert] = "Student has reservations and can't be removed."
+      @student = Student.new
```

Comment:
again, is this relevant to this PR for updating the ldap_lookup gem? If not it should be in a seprate PR

Reply:
Yes, this is relevant. The PR updates the page where a program's student list is updated. The page allows you to add and remove students. If a student can't be removed (because it has reservations) the input field for a new student should be empty. Without @student = Student.new the uniqname of the student (that couldn't be removed) appears in the input field.